zanshincli/main.py

```diff
@@ -998,6 +997,25 @@ def organization_scan_target_scan_get(
     client = Client(profile=global_options['profile'])
     dump_json(client.get_organization_scan_target_scan(organization_id, scan_target_id, scan_id))
 
+###################################################
+# Scan Target Groups App
+###################################################
+
+scan_target_group_app = typer.Typer()
+organization_app.add_typer(scan_target_group_app, name="scan-target-groups",
+                   help="Operations on organizations scan target groups the API key owner has direct access to")
+
+
+@scan_target_group_app.command(name='script')
+def scan_target_groups_script(
+        organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
+        scan_target_group_id: UUID = typer.Argument(..., help="UUID of the scan target group")    
+):
+    """
+    Get download URL of the scan target group.
```

Comment:
What does this return? An URL? Is this the Oracle terraform script?

Reply:
"Get the terraform download URL of the scan target group" is what is written in the documentation, should I change something in the docstring?

Reply:
That is fine.

I think it's going to be "Gets" though.